src/context.ts

```diff
@@ -7,7 +7,7 @@ export interface Inputs {
   logout: string;
 }
 
-export async function getInputs(): Promise<Inputs> {
+export function getInputs(): Inputs {
```

Comment:
The `getInput` isn't async so I didn't think we needed a promise here and it helps to bubble up the errors that get thrown.